src/wallpapers/linux.rs

```diff
@@ -0,0 +1,177 @@
+use super::Desktop;
+use std::error::Error;
+use std::io::BufRead;
+use std::path::PathBuf;
+use std::process::Command;
+
+/// A desktop environment
+#[derive(Debug, Copy, Clone, Eq, PartialEq)]
+pub enum DesktopEnvt {
+    Gnome,
+    Cinnamon,
```

Comment:
Just being pedantic -- change the types to correct lexical terms. Such as ```GNOME``` and ```KDE```.